modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/index.js

```diff
@@ -57,6 +59,38 @@ const TeamMembers = () => {
 					</Button>
 				</div>
 			</div>
+			{statusActivedDXPC && (
+				<div className="bg-brand-primary-lighten-6 border-0 card card-flat cp-manager-product-container mt-5">
+					<div className="p-4">
+						<p className="h4">Manage Product Users</p>
+
+						<p className="mt-2 text-neutral-7 text-paragraph-sm">
+							Manage roles and permissions of users within each
+							product.
+						</p>
+
+						<div className="d-flex">
+							<Button
+								appendIcon="shortcut"
+								className="align-items-stretch btn btn-ghost cp-manager-product-button d-flex mr-3 p-2 text-neutral-10"
+							>
+								<p className="font-weight-semi-bold h6 m-0 pl-1">
+									Manage DXP Cloud Users
+								</p>
+							</Button>
+
+							<Button
+								appendIcon="shortcut"
+								className="align-items-stretch btn btn-ghost cp-manager-product-button d-flex p-2 text-neutral-10"
+							>
+								<p className="font-weight-semi-bold h6 m-0 pl-1">
+									Manage Analytics Cloud Users
+								</p>
+							</Button>
+						</div>
+					</div>
+				</div>
+			)}
```

Comment:
Hey @Lucas2691, how about creating a component to this? 
I think it would be better to create this in components folder level and import the component here

Reply:
Yes @iagochaves, I will do this!